apps/dialtone-documentation/docs/about/whats-new/posts/2024-4-11.md

```diff
@@ -0,0 +1,282 @@
+---
+heading: Added Tree-shaking support for dialtone-vue and dialtone-icons
+author: Tico Ortega
+posted: '2024-4-11'
+---
+
+<BlogPost :author="$frontmatter.author" :posted="parse($frontmatter.posted, 'y-M-d', new Date())" :heading="$frontmatter.heading">
+
+# TL;DR:
+
+- Improved dialtone-vue **bundle size**.
+- Added **tree-shaking** support for dialtone-vue and dialtone icons.
+- **Updated mono-package exports** (migration guide included)
+- Now it's possible to **use icons from @dialpad/dialtone-icons directly**
+
+___
+
+**👋 Hey there,** we're excited to share some updates regarding the bundle size of dialtone-vue,
+how to migrate to latest dialtone version as the exports changed and how can you use @dialpad/dialtone-icons library to
+use isolated icons to avoid bundling a bunch of icons you're not using.
+
+## Dialtone-vue bundle size
+
+We've improved the dialtone-vue bundle size by externalizing all the dependencies that are not part of the library.
+
+### Before
+
+![dialtone vue bundle analysis before](https://private-user-images.githubusercontent.com/87546543/314393856-615e82cb-5fb4-4529-a343-14fb1ccde2f6.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTI4NTM2MjEsIm5iZiI6MTcxMjg1MzMyMSwicGF0aCI6Ii84NzU0NjU0My8zMTQzOTM4NTYtNjE1ZTgyY2ItNWZiNC00NTI5LWEzNDMtMTRmYjFjY2RlMmY2LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA0MTElMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNDExVDE2MzUyMVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTJiMWYzZjAyOWE5NGE4NGQ0MjEzNjRjNzRjNDgxOTA4ZWUyYmM2MzRmYzg3ODAzZDA3MjI3YjYyZjNlMzg4Y2MmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.3nw8E5cqq0ANcqwAQ_ycHnYmJGR9Tm2cSnh8yvOIgfs)
+![dialtone vue bundle size before](https://private-user-images.githubusercontent.com/87546543/314394019-84056a1f-22e3-46df-bacf-64384af66f56.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTI4NTM2MjEsIm5iZiI6MTcxMjg1MzMyMSwicGF0aCI6Ii84NzU0NjU0My8zMTQzOTQwMTktODQwNTZhMWYtMjJlMy00NmRmLWJhY2YtNjQzODRhZjY2ZjU2LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA0MTElMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNDExVDE2MzUyMVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWNlNTNlOGE0ZDQzOTdmMzljN2VkN2RmM2E0OGU3MzIyNDE3ZGQzNWFiN2ZmYjE5NTdjMmZkMjEzNjI1ZTllMTcmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.uFTEwDebpu3EUQeE22rW2rafQfsSauFJLaDDV1dWtrY)
```

Comment:
What's up with these images? they aren't displaying and have a weird address

Reply:
Those were images from the #216 PR, downloaded and added images to our repo to avoid issues with those links.